requirements.txt

```diff
@@ -1,4 +1,4 @@
-nanopub==0.2.2
+git+https://github.com/fair-workflows/nanopub.git@use-test-endpoint
```

Comment:
Should point to new released nanopub version when https://github.com/fair-workflows/nanopub/pull/25 is merged

Reply:
Nanopub PR 25 is now approved with a couple of comments.